apps/test/unit/applab/ExporterTest.js

```diff
@@ -75,7 +75,7 @@ describe('The Exporter,', function () {
     });
   });
 
-  describe("when exporting,", function () {
+  xdescribe("when exporting,", function () {
```

Comment:
Might be worth a short comment explaining that they're disabled intentionally. Or even just deleting until they're ready?